keyboards/hs60/rules.mk

```diff
@@ -69,6 +69,8 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 RGB_MATRIX_ENABLE = yes     # Use RGB matrix
 
+LAYOUTS = 60_ansi 60_iso
```

Comment:
we include 60_iso here, is there a HS60_ISO that needs to be defined as well?

Reply:
@mechmerlin the HS60_ANSI is specified so that the RGB Matrix knows which lights should be used. 

I'm not sure if the is a way to tell in the makefile stuff.  I can dig.  But from what I can tell, it should support both 60% ANSI and ISO.